src/components/containers/victory-voronoi-container.js

```diff
@@ -1,8 +1,9 @@
 import PropTypes from "prop-types";
 import React from "react";
+import { defaults, isFunction } from "lodash";
 import { VictoryContainer, VictoryTooltip, Helpers, TextSize } from "victory-core";
+import { omit } from "victory-core/es/victory-util/helpers";
```

Comment:
It bugs me that we have just this one use of `omit` left outside of victory-core. What do you think about refactoring it to use a different method all together?

Reply:
love it!